### PR TITLE
img2txt: add page

### PR DESCRIPTION
**Pull request by sakibjunaed**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by EmilySeville7cfg — APPROVED** [review]
- No unmentioned mandatory fixes found, approved.
- This page should be reviewed by `img2txt` users.
- Thanks for a contribution! ❤️

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/img2txt.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# img2txt
+
+> Convert images to colour ASCII characters and output them to text-based coloured files.
+> More information: <https://manned.org/img2txt.1>.
+- Change output column count:
```
[the remark this answers is not in the document]
Comment (EmilySeville7cfg) [reply pages/linux/img2txt.md]: Fixed by PR author.

File: pages/linux/img2txt.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`img2txt --width={{width}}`
```
[the remark this answers is not in the document]
Comment (EmilySeville7cfg) [reply pages/linux/img2txt.md]: Fixed by PR author.

File: pages/linux/img2txt.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# img2txt
+
+> Convert images to colour ASCII characters and output them to text-based coloured files.
+> More information: <https://manned.org/img2txt.1>.
```
[the remark this answers is not in the document]
Comment (EmilySeville7cfg) [reply pages/linux/img2txt.md]: Fixed by PR author.

**Comment by kbdharun** [comment]
Yay, I this PR was my 1000th review in `tldr` lol 🎉 .

**Pull request merged**